TLM/TLM/UI/SubTools/TTL/TimedTrafficLightsTool.cs

```diff
@@ -166,146 +164,146 @@ public TimedTrafficLightsTool(TrafficManagerTool mainTool)
                     return;
                 }
                 RefreshCurrentTimedNodeIds(HoveredNodeId);
-                this.SetToolMode(TTLToolMode.ShowLights);
+                SetToolMode(TTLToolMode.ShowLights);
             }
 
             TrafficLightSimulationManager tlsMan = TrafficLightSimulationManager.Instance;
 
             switch (ttlToolMode_) {
                 case TTLToolMode.SelectNode:
                 case TTLToolMode.ShowLights: {
-                    if (ttlToolMode_ == TTLToolMode.ShowLights) {
-                        this.SetToolMode(TTLToolMode.SelectNode);
-                        ClearSelectedNodes();
-                    }
-
-                    if (!tlsMan.HasTimedSimulation(HoveredNodeId)) {
-                        if (IsNodeSelected(HoveredNodeId)) {
-                            RemoveSelectedNode(HoveredNodeId);
-                        } else {
-                            AddSelectedNode(HoveredNodeId);
+                        if (ttlToolMode_ == TTLToolMode.ShowLights) {
+                            SetToolMode(TTLToolMode.SelectNode);
```

Comment:
[Note] I think TMPE policy is that using this is recommended. I don't care either way

Reply:
This was Visual Studio doing things I didn't ask it to. :-(